configs/node-collector/journal/filter-tagging.conf

```diff
@@ -0,0 +1,6 @@
+<filter **>
+  @type record_transformer
+  <record>
+    tags "ADD-SPACE-DELIMITED-TAGS-HERE"
```

Comment:
Do we need tags specified both here and in the catchall conf file of the same name?